executor/test/showtest/show_test.go

```diff
@@ -2082,3 +2082,48 @@ func TestShowBindingDigestField(t *testing.T) {
 	result = tk.MustQuery("show global bindings;")
 	require.Equal(t, len(result.Rows()), 0)
 }
+
+func TestShowPasswordVariable(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+
+	tk.MustExec("SET GLOBAL authentication_ldap_sasl_bind_root_pwd = ''")
+	rs, err := tk.Exec("show variables like 'authentication_ldap_sasl_bind_root_pwd'")
+	require.NoError(t, err)
+	require.Equal(t, tk.ResultSetToResult(rs, "").Rows()[0][1], "")
+	rs, err = tk.Exec("SELECT current_value FROM information_schema.variables_info WHERE VARIABLE_NAME LIKE 'authentication_ldap_sasl_bind_root_pwd'")
+	require.NoError(t, err)
+	require.Equal(t, tk.ResultSetToResult(rs, "").Rows()[0][0], "")
+
+	tk.MustExec("SET GLOBAL authentication_ldap_sasl_bind_root_pwd = password")
+	defer func() {
+		tk.MustExec("SET GLOBAL authentication_ldap_sasl_bind_root_pwd = ''")
+	}()
+	rs, err = tk.Exec("show variables like 'authentication_ldap_sasl_bind_root_pwd'")
+	require.NoError(t, err)
+	require.Equal(t, tk.ResultSetToResult(rs, "").Rows()[0][1], "******")
+	rs, err = tk.Exec("SELECT current_value FROM information_schema.variables_info WHERE VARIABLE_NAME LIKE 'authentication_ldap_sasl_bind_root_pwd'")
+	require.NoError(t, err)
+	require.Equal(t, tk.ResultSetToResult(rs, "").Rows()[0][0], "******")
+
+	tk.MustExec("SET GLOBAL authentication_ldap_simple_bind_root_pwd = ''")
+	rs, err = tk.Exec("show variables like 'authentication_ldap_simple_bind_root_pwd'")
+	require.NoError(t, err)
+	require.Equal(t, tk.ResultSetToResult(rs, "").Rows()[0][1], "")
+	rs, err = tk.Exec("SELECT current_value FROM information_schema.variables_info WHERE VARIABLE_NAME LIKE 'authentication_ldap_simple_bind_root_pwd'")
+	require.NoError(t, err)
+	require.Equal(t, tk.ResultSetToResult(rs, "").Rows()[0][0], "")
+
+	tk.MustExec("SET GLOBAL authentication_ldap_simple_bind_root_pwd = password")
+	defer func() {
+		tk.MustExec("SET GLOBAL authentication_ldap_simple_bind_root_pwd = ''")
+	}()
+
+	rs, err = tk.Exec("show variables like 'authentication_ldap_simple_bind_root_pwd'")
+	require.NoError(t, err)
+	require.Equal(t, tk.ResultSetToResult(rs, "").Rows()[0][1], "******")
+	rs, err = tk.Exec("SELECT current_value FROM information_schema.variables_info WHERE VARIABLE_NAME LIKE 'authentication_ldap_simple_bind_root_pwd'")
+	require.NoError(t, err)
+	require.Equal(t, tk.ResultSetToResult(rs, "").Rows()[0][0], "******")
+
```

Comment:
We have the following error in CI system(https://do.pingcap.net/jenkins/blue/organizations/jenkins/pingcap%2Ftidb%2Fghpr_build/detail/ghpr_build/28794/pipeline):

```
compilepkg: nogo: errors found by nogo during build-time code analysis:
executor/test/showtest/show_test.go:2086:45: empty-lines: extra empty line at the end of a block (all_revive)
```

Reply:
Please remove the blank line, thanks.